File: python-client/goredis/strings.py
from .client import get_client

def Get(key):
    """Retrieve the string value of a key."""
    return get_client().send_command("GET", key)

def Set(key, value):
    """Set the string value of a key."""
    return get_client().send_command("SET", key, value)

def Incr(key):
    """Increment the integer value of a key by one."""
    return get_client().send_command("INCR", key)

def Decr(key):
    """Decrement the integer value of a key by one."""
    return get_client().send_command("DECR", key)

def IncrBy(key, increment):
    """Increment the integer value of a key by the given amount."""
    return get_client().send_command("INCRBY", key, increment)

def DecrBy(key, decrement):
    """Decrement the integer value of a key by the given amount."""
    return get_client().send_command("DECRBY", key, decrement)

def MGet(*keys):
    """Get the values of all the given keys."""
    return get_client().send_command("MGET", *keys)

def MSet(**mapping):
    """Set multiple keys to multiple values using a dictionary or keyword arguments."""
    args = []
    for k, v in mapping.items():
        args.extend([k, v])
    return get_client().send_command("MSET", *args)

def StrLen(key):
    """Get the length of the string value stored at key."""
    return get_client().send_command("STRLEN", key)

File: go-client/strings.go
package goredis

// Get retrieves the value of a key.
func Get(key string) (interface{}, error) {
	return mustGetClient().SendCommand("GET", key)
}

// Set sets the value of a key.
func Set(key string, value interface{}) (interface{}, error) {
	return mustGetClient().SendCommand("SET", key, value)
}

// Incr increments the integer value of a key by one.
func Incr(key string) (interface{}, error) {
	return mustGetClient().SendCommand("INCR", key)
}

// Decr decrements the integer value of a key by one.
func Decr(key string) (interface{}, error) {
	return mustGetClient().SendCommand("DECR", key)
}

// IncrBy increments the integer value of a key by the given increment.
func IncrBy(key string, increment int) (interface{}, error) {
	return mustGetClient().SendCommand("INCRBY", key, increment)
}

// DecrBy decrements the integer value of a key by the given decrement.
func DecrBy(key string, decrement int) (interface{}, error) {
	return mustGetClient().SendCommand("DECRBY", key, decrement)
}

// MGet retrieves the values of multiple keys.
func MGet(keys ...string) (interface{}, error) {
	cmdArgs := append([]interface{}{"MGET"}, toInterfaceSlice(keys)...)
	return mustGetClient().SendCommand(cmdArgs...)
}

// MSet sets multiple keys to their respective values.
func MSet(mapping map[string]interface{}) (interface{}, error) {
	cmdArgs := []interface{}{"MSET"}
	for k, v := range mapping {
		cmdArgs = append(cmdArgs, k, v)
	}
	return mustGetClient().SendCommand(cmdArgs...)
}

// StrLen returns the length of the string value stored at key.
func StrLen(key string) (interface{}, error) {
	return mustGetClient().SendCommand("STRLEN", key)
}
